language-support/ts/daml-ledger/index.ts

```diff
@@ -196,9 +200,7 @@ export type Query<T> = T extends object ? {[K in keyof T]?: Query<T[K]>} : T;
  * queries. As long as that restriction stays, there is no need for any kind of
  * encoding here.
  */
-function encodeQuery<T extends object, K, I extends string>(template: Template<T, K, I>, query?: Query<T>): unknown {
-  // I could not get the "unused" warning silenced, but this seems to count as "used"
-  [template];
+function encodeQuery<T extends object, K, I extends string>(_template: Template<T, K, I>, query?: Query<T>): unknown {
```

Comment:
Fairly sure I'd tried that 🤔. Oh well. Thanks for fixing!

Reply:
We may have changed the compiler version and/or configuration in the meantime, not sure.